### PR TITLE
addBookFromPathAndGetId unit test

### PR DESCRIPTION
**Pull request by jetownfeve21**

### CODE REVIEW

File: test/manager.cpp [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+TEST(ManagerTest, addBookFromPathAndGetIdTest)
+{
+    kiwix::Library lib;
+    kiwix::Manager manager = kiwix::Manager(&lib);
+
+    auto bookId = manager.addBookFromPathAndGetId("./test/example.zim");
+    EXPECT_NE(bookId, "");
+    kiwix::Book book = lib.getBookById(bookId);
+    EXPECT_NE(book.getPath(), "");
```
Comment (mgautierfr) [inline test/manager.cpp:16]: Wouldn't be better to check that `book.getPath()` equals "./test/example.zim" ?

File: test/manager.cpp [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    
+    const std::string pathToSave = "./pathToSave";
+    const std::string url = "url";
+    bookId = manager.addBookFromPathAndGetId("./test/example.zim", pathToSave, url, true);
+    book = lib.getBookById(bookId);
+    auto savedPath = isRelativePath(pathToSave) ?
```
Comment (mgautierfr) [inline test/manager.cpp:22]: You already know that `pathToSave` is relative.
And by the way, you should not use the same code that the code you want to test to compute the right result.

**Comment by legoktm** [comment]
@jetownfeve21 @mgautierfr: What is the source/copyright on the `test/data/example.zim` file that was added?

**Comment by kelson42** [comment]
@jetownfeve21 Except for the logo of wikibooks, this ZIM files contains two empty pages.

**Comment by jetownfeve21** [comment]
@legoktm @kelson42  this zim file only exists for the purpose of the unit tests

**Pull request merged**